### PR TITLE
Revert "Add Querier support for Graphs"

### PR DESCRIPTION
**Pull request by muskankhedia**
Reverts bench-routes/bench-routes#364

### CODE REVIEW

**Comment by ruddi10** [comment]
@muskankhedia  is that working slow even on your local setup

**Comment by muskankhedia** [comment]
@ruddi10 It is working fine with a smaller set of data but with a larger set of data, the entire tab(bench-routes dashboard) starts lagging. As when we have data for say of at least 4-5days, then due to the for loops used at the frontend to distinguish the data for different graphs the UI starts lagging. This is why we plan to revert it for now.

**Comment by ruddi10** [comment]
> @ruddi10 It is working fine with a smaller set of data but with a larger set of data, the entire tab(bench-routes dashboard) starts lagging. As when we have data for say of at least 4-5days, then due to the for loops used at the frontend to distinguish the data for different graphs the UI starts lagging. This is why we plan to revert it for now.

@muskankhedia like even if data is for 4-5 days it will work fine unless we try to query it I guess

**Comment by muskankhedia** [comment]
> > @ruddi10 It is working fine with a smaller set of data but with a larger set of data, the entire tab(bench-routes dashboard) starts lagging. As when we have data for say of at least 4-5days, then due to the for loops used at the frontend to distinguish the data for different graphs the UI starts lagging. This is why we plan to revert it for now.
> 
> @muskankhedia like even if data is for 4-5 days it will work fine unless we try to query it I guess

Yes, but if there is a feature enabled we can't expect for everyone to not use it.

**Comment by ruddi10** [comment]
> > @ruddi10 It is working fine with a smaller set of data but with a larger set of data, the entire tab(bench-routes dashboard) starts lagging. As when we have data for say of at least 4-5days, then due to the for loops used at the frontend to distinguish the data for different graphs the UI starts lagging. This is why we plan to revert it for now.
> 
> @muskankhedia like even if data is for 4-5 days it will work fine unless we try to query it I guess



> > > @ruddi10 It is working fine with a smaller set of data but with a larger set of data, the entire tab(bench-routes dashboard) starts lagging. As when we have data for say of at least 4-5days, then due to the for loops used at the frontend to distinguish the data for different graphs the UI starts lagging. This is why we plan to revert it for now.
> > 
> > 
> > @muskankhedia like even if data is for 4-5 days it will work fine unless we try to query it I guess
> 
> Yes, but if there is a feature enabled we can't expect everyone to not use it.

No no I mean think that querying 5 days data might be a slow process thats why this is happening as for the issue of the freezing the other person mentioned that might be because he havent run make run

**Review by Harkishen-Singh — APPROVED** [review]

**Comment by muskankhedia** [comment]
> > > @ruddi10 It is working fine with a smaller set of data but with a larger set of data, the entire tab(bench-routes dashboard) starts lagging. As when we have data for say of at least 4-5days, then due to the for loops used at the frontend to distinguish the data for different graphs the UI starts lagging. This is why we plan to revert it for now.
> > 
> > 
> > @muskankhedia like even if data is for 4-5 days it will work fine unless we try to query it I guess
> 
> > > > @ruddi10 It is working fine with a smaller set of data but with a larger set of data, the entire tab(bench-routes dashboard) starts lagging. As when we have data for say of at least 4-5days, then due to the for loops used at the frontend to distinguish the data for different graphs the UI starts lagging. This is why we plan to revert it for now.
> > > 
> > > 
> > > @muskankhedia like even if data is for 4-5 days it will work fine unless we try to query it I guess
> > 
> > 
> > Yes, but if there is a feature enabled we can't expect everyone to not use it.
> 
> No no I mean think that querying 5 days data might be a slow process thats why this is happening as for the issue of the freezing the other person mentioned that might be because he havent run make run

I tried in my local, and I have started the backend server and faced the same issue. And suppose if someone haven't started the golang server, they will receive the alert `Unable to reach the service: error` instead of the system starting to lag.

**Review by aquibbaig — APPROVED** [review]

File: dashboard/v1.1/package.json [hunk 1]
```diff
@@ -9,10 +9,8 @@
     "apexcharts": "^3.19.0",
     "material-table": "1.68.0",
     "material-ui-search-bar": "^1.0.0",
-    "moment": "^2.29.1",
```
Comment (chetanyakan) [inline dashboard/v1.1/package.json:12]: Either way `moment` is deprecated. We should use other alternatives like `day.js`.

Refer: https://momentjs.com/docs/#/-project-status/

Comment (ruddi10) [reply dashboard/v1.1/package.json:12]: @chetanyakan react-datetime has a dependency on momentjs

**Pull request closed without merging**